source/grid/grid_generator.cc

```diff
@@ -7138,6 +7138,103 @@ namespace GridGenerator
          {{{{0, 1, 12}}, {{1, 12, 3}}, {{12, 3, 2}}, {{0, 12, 2}}}},
          {{{{4, 5, 13}}, {{5, 13, 7}}, {{13, 7, 6}}, {{4, 13, 6}}}}}};
 
+    static const std::array<std::array<unsigned int, 2>, 8>
+      table_2D_inner_faces = {{{{6, 4}},
```

Comment:
Can you add a couple of lines of comments in front of each of these tables? I have no idea what these variables mean -- presumably, they *map* something *to* something, but it's not clear what it is. I think it would be useful to name these variables better (e.g. `boundary_face_to_cell_map_2d` or similar, assuming that's what this is: in other words, a `X_to_Y_map` name). It would also be good if each had a description before its declaration that illustrates what the indices actually are. This one, for example, is a table with two indices, but what are they? Face numbers? Vertex numbers? Of quads? Of triangles?

Reply:
I haven't changed the names of these tables yet, because I personally think `X_to_Y_map` is a little misleading. How about: `vertices_id_boundary_faces_3d`,   `vertices_id_inner_edge_3d` and so on? Because the table contains the IDs of the vertices that form e.g. a face on the boundary or an edge that is inside.

Reply:
How about `vertex_ids_for_boundary_faces` as a name?

Reply:
that sounds good to me, assuming we'll additionally add _2d or _3d, respectively.

Reply:
Yes, the suffix is fine of course.